ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplDumpTask.java

```diff
@@ -742,7 +845,7 @@ long currentNotificationId(Hive hiveDb) throws TException {
 
   Path dumpDbMetadata(String dbName, Path metadataRoot, long lastReplId, Hive hiveDb) throws Exception {
     // TODO : instantiating FS objects are generally costly. Refactor
-    Path dbRoot = getBootstrapDbRoot(metadataRoot, dbName, false);
```

Comment:
This was different for bootstrap duing incremental flow
`if (isIncrementalPhase) {
      dumpRoot = new Path(dumpRoot, ReplUtils.INC_BOOTSTRAP_ROOT_DIR_NAME);
    }`
How is the path now?